src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="connection.driver_class">com.mysql.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/internet_shop?serverTimezone=UTC</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">1</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">validate</property>
+        <mapping class="mate.academy.internetshop.model.Item" />
+    </session-factory>
+</hibernate-configuration>
```

Comment:
empty line at the EOF

Reply:
Added  line. Wrote wrong comment for commit